fv3core/pace/fv3core/stencils/remapping.py

```diff
@@ -636,7 +634,7 @@ def __call__(
 
         self._copy_from_below_stencil(ua, pe)
         dtmp = 0.0
-        if last_step and not do_adiabatic_init:
+        if last_step:
```

Comment:
The reason I removed do_adiabatic_init is because it’s not available at initialization time, it’s actually a call-time argument in the Fortran. To make it call-time in pace requires refactoring all call interfaces between the front of the dycore and where it’s used. Also, I chatted with @elynnwu and we concluded it’s dangerous to include code pathways that aren’t regression tested. do_adiabatic_init very much changes what the dycore is supposed to do.